app/layout/statics/button-link/component.tsx

```diff
@@ -18,23 +19,23 @@ export const StaticButtonLink: React.FC<StaticButtonLinkProps> = ({
 }: StaticButtonLinkProps) => {
   const children = (
     <>
-      <p className="mr-4 text-lg text-primary-500 hover:underline">{caption}</p>
-      <div className="flex items-center justify-center bg-transparent border rounded-full h-7 w-7 border-primary-500">
-        <Icon icon={ARROW_RIGHT_2_SVG} className="w-3 h-3 text-primary-500" />
+      <p className="w-48 mr-4 text-lg text-gray-600 hover:text-gray-800 underline">{caption}</p>
```

Comment:
I don't think adding a fixed width is the best option, If you put a maxWidth is better

Reply:
I'll keep this to follow design. We'll change it if we need to.

Reply:
I'll keep this to follow design. We'll change it if we need to. (is this comment duplicated?)